controllers/chat/ChatController.js

```diff
@@ -13,6 +13,11 @@ const BetterSocialConstantListUtils = require('../../services/bettersocial/const
 const {CHANNEL_TYPE} = require('../../helpers/constants');
 const ChatAnonUserInfoFunction = require('../../databases/functions/chatAnonUserInfo');
 const BetterSocialCore = require('../../services/bettersocial');
+const {
+  is_all_anon_user,
+  handle_anon_to_anon_channel_owner,
+  handle_anon_to_anon_channel_member
+} = require('../../services/bettersocial/chat/allAnonChat');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New functions have been imported from the `../../services/bettersocial/chat/allAnonChat` module. These functions are used later in the code to handle anonymous-to-anonymous chat scenarios.
```diff
+ const {
+   is_all_anon_user,
+   handle_anon_to_anon_channel_owner,
+   handle_anon_to_anon_channel_member
+ } = require('../../services/bettersocial/chat/allAnonChat');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->